main.go

```diff
@@ -37,11 +37,25 @@ func main() {
 						error := errorConstructor.New(err.Error())
 						reject.Invoke(error)
 					} else {
-						var stmts []interface{}
-						for _, stmt := range ast.Statements {
-							stmts = append(stmts, stmt.String())
+						response := make(map[string]interface{})
+						statements := make([]interface{}, len(ast.Statements))
```

Comment:
To make tinygo compiler happy with WASM, our collections always have to be collections of anon interfaces.